Makefile

```diff
@@ -139,6 +139,8 @@ helm-uninstall: ; $(info $(M)...Begin to helm-uninstall.)  @ ## Helm-uninstall.
 # Run tests
 test: vet ;$(info $(M)...Begin to run tests.)  @ ## Run tests.
 	export KUBEBUILDER_CONTROLPLANE_START_TIMEOUT=2m; go test ./pkg/... ./cmd/... -covermode=atomic -coverprofile=coverage.txt
+	cd staging/src/kubesphere.io/api ; GOFLAGS="" go test ./...
+	cd staging/src/kubesphere.io/client-go ; GOFLAGS="" go test ./...
```

Comment:
I think the logic is similar to: https://github.com/kubesphere/kubesphere/blob/master/hack/update-vendor.sh#L124-L134

Reply:
Do you mean to refactor the commands with a shell?

Reply:
Another staging library may be added, should not be hard code here.

Reply:
Could we merge this pr first? We can refactor the commands with another pr. Some work has been blocked because of this pr.

Reply:
Any more comments? @wansir